org.firestormviewer.FirestormViewer.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id":"org.firestormviewer.FirestormViewer",
+  "branch":"stable",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"19.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"firestorm-viewer",
+  "separate-locales":false,
+  "finish-args":[
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--allow=multiarch",
+    "--device=dri",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications",
+    "--own-name=com.secondlife.ViewerAppAPIService",
+    "--filesystem=xdg-run/app/com.discordapp.Discord:create"
+  ],
+  "cleanup":[
+    "/share/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/lib/pkgconfig",
+    "/lib/debug"
+  ],
+  "build-options":{
+    "strip":true
+  },
+  "add-extensions":{
+    "org.freedesktop.Platform.Compat.i386":{
+      "directory":"lib/i386-linux-gnu",
+      "version":"19.08"
+    }
+  },
+  "sdk-extensions":[
+    "org.freedesktop.Sdk.Compat.i386",
+    "org.freedesktop.Sdk.Extension.toolchain-i386"
+  ],
+  "modules":[
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/gtk2/gtk2.json",
+    "shared-modules/dbus-glib/dbus-glib-0.110.json",
+    "modules/wine.json",
+    "modules/libnotify.json",
+    "modules/gamemode.json",
+    {
+      "name":"firestorm",
+      "buildsystem":"simple",
+      "cleanup":"viewer.tar.xz",
+      "sources":[
+        {
+          "type":"file",
+          "dest-filename":"viewer.tar.xz",
+          "url":"http://downloads.firestormviewer.org/linux/Phoenix_Firestorm-Releasex64_x86_64_6.3.2.58052.tar.xz",
+          "sha256":"c80e16541697aa32925781b28cd58545645cb5a2a814d5f296a39a73a03d38c1"
+        },
+        {
+          "type":"file",
+          "path": "build.sh"
+        },
+        {
+          "type":"file",
+          "path":"assets/firestorm-viewer"
+        },
+        {
+          "type":"file",
+          "path":"assets/launch_url.sh"
+        },
+        {
+          "type":"file",
+          "path":"assets/org.firestormviewer.FirestormViewer.desktop"
+        },
+        {
+          "type":"file",
+          "path":"assets/org.firestormviewer.FirestormViewer.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html